Automattic/portkey-android
# Changed file path for Uri as source for mp4composer

This PR makes mp4composer able to process both local video files as external video files provided by the media picker through a `Uri`

To test:
Local file:
1. open the app, try capturing a video with the video capture control (tap and hold)
2. create and save a new video with emoji, text, etc.

Media provided through media picker (content provider):
1. open the app, and tap on the image gallery icon, located to the left of the video/image capture control
2. select a video (preferably a short one for the test)
3. add emoji / text and save
4. confirm the new video has been saved (either by opening the Photos app, or by tapping on the gallery icon again, the new video should be the first one in the media picker).
